THIRD-PARTY-PROGRAMS

```diff
@@ -303,6 +303,9 @@ CMake was initially developed by Kitware with the following sponsorship:
 6. Xbyak_aarch64 (third_party/xbyak_aarch64/)
 Copyright 2019-2020 FUJITSU LIMITED
 
+7. Khronos OpenCL Headers (third_party/CL/)
```

Comment:
New component also needs to be mentioned in [License section](https://github.com/uxlfoundation/oneDNN?tab=readme-ov-file#license) of README.md